renderdoc/driver/vulkan/vk_core.h

```diff
@@ -444,6 +444,26 @@ class WrappedVulkan : public IFrameCapturer
   bool m_ExtendedDynState2 = false;
   bool m_ExtendedDynState2Logic = false;
   bool m_ExtendedDynState2CPs = false;
+  bool m_ExtendedDynState3TesselDomain = false;
+  bool m_ExtendedDynState3DepthClampEnable = false;
+  bool m_ExtendedDynState3PolyMode = false;
+  bool m_ExtendedDynState3RastSamples = false;
+  bool m_ExtendedDynState3SampleMask = false;
+  bool m_ExtendedDynState3AlphaToCover = false;
+  bool m_ExtendedDynState3AlphaToOne = false;
+  bool m_ExtendedDynState3LogicEnable = false;
+  bool m_ExtendedDynState3CBEnable = false;
+  bool m_ExtendedDynState3CBEquation = false;
+  bool m_ExtendedDynState3WriteMask = false;
+  bool m_ExtendedDynState3RastStream = false;
+  bool m_ExtendedDynState3ConservRast = false;
+  bool m_ExtendedDynState3PrimOverest = false;
+  bool m_ExtendedDynState3DepthClip = false;
+  bool m_ExtendedDynState3SampleLoc = false;
+  bool m_ExtendedDynState3ProvokingVert = false;
+  bool m_ExtendedDynState3LineRast = false;
+  bool m_ExtendedDynState3LineStipple = false;
+  bool m_ExtendedDynState3DepthClipNeg = false;
```

Comment:
It makes me sad that the ext had this many toggles in it :(.

Aside from the other comments about removing the entries here that correspond to unsupported extensions, this is fine. I think I will try and figure out a way to tidy this up though as this is becoming silly.